### PR TITLE
GH-1528: Release 0.11.0

### PR DESCRIPTION
**Pull request by tbl3rd**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-1528

### Changes

- https://github.com/broadinstitute/wfl/blob/a8611c90f27868f31f64e325b7cfb50eb5c3b573/CHANGELOG.md?plain=1#L1-L8

### Review Instructions

```
tbl@wmf05-d86 ~/Broad/wfl # time make api TARGET=system
...
Ran 32 tests containing 367 assertions.
0 failures, 0 errors.
api system finished on Wed Dec 15 11:19:23 EST 2021

real    16m6.139s
user    2m16.943s
sys     0m16.354s
tbl@wmf05-d86 ~/Broad/wfl #
```

### CODE REVIEW

**Comment by okotsopoulos** [comment]
Do you mean to merge this into `main`?  (Sorry if I am missing something!)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Release 0.11.0
```
Comment (okotsopoulos) [inline CHANGELOG.md:1]: Changelog looks good, thank you.  Pretty small release!

Comment (tbl3rd) [reply CHANGELOG.md:1]: Still too big, I think.

**Review by okotsopoulos — APPROVED** [review]
Thanks. Approving assuming that system tests are passing.

**Pull request merged**